lib/src/values/logic_values.dart

```diff
@@ -27,8 +27,8 @@ import 'package:rohd/rohd.dart';
 class _SmallLogicValues extends LogicValues {
   // Each 0/1 bit in value is 0/1 if !invalid, else is x/z
 
-  final int _value;
```

Comment:
these bug fixes have been pushed into main, you can pull from main so these don't show up as a diff in this PR